mail_compose_message_reply_to/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Mail Compose Message Reply To",
+    "category": "Accouting",
```

Comment:
```suggestion
    "category": "Discuss",
```
Following the category of the mail module.